C2.md

```diff
@@ -80,8 +81,10 @@ be requested via C2 DESCRIBE manifest command.
     nifi.c2.rest.listener.cacert=<SSL Cert path>
 
     # specify the rest URIs if using RESTSender
-    nifi.c2.rest.url=http://<your-c2-server>/<c2-api-path>/c2-protocol/heartbeat
-    nifi.c2.rest.url.ack=http://<your-c2-server>/<c2-api-path>/c2-protocol/acknowledge
+    nifi.c2.rest.path.base=https://<your-c2-server>/<c2-api-path>
+    # specify either absolute url or relative to the nifi.c2.rest.path.base url for hearbeat and acknowledge
+    nifi.c2.rest.url=/c2-protocol/heartbeat
+    nifi.c2.rest.url.ack=/c2-protocol/acknowledge
```

Comment:
I think these shouldn't include a leading slash, because they are relative paths. That is unless MiNiFi Java works the same strange way. Wdyt?

Reply:
this is how it's done in minifi java